source/Preferences.h

```diff
@@ -28,6 +28,12 @@ class Preferences {
 		adaptive,
 	};
 
+	enum class BoardingPriority : int {
+		distance = 0,
```

Comment:
Do we also want to use proximity, value or mixed here? (Similar to what we use in the preferences file?)

Reply:
Would make sense to do so you're right